rmw_zenoh_cpp/src/detail/rmw_context_impl_s.hpp

```diff
@@ -28,12 +28,8 @@
 #include "rmw/ret_types.h"
 #include "rmw/types.h"
 
-#ifdef _MSC_VER
-#pragma warning(push)
-#pragma warning(disable: 4099)
-#endif
 ///=============================================================================
-class rmw_context_impl_s final
+struct rmw_context_impl_s final
```

Comment:
@clalancette was this change to struct intentional?

Reply:
There is a warning on Windows, `rmw_context_impl_s` is defined in the `rmw` as a struct but we are implementing this as a class.

Reply:
I see. Thanks for pointing that out.